parltrack_meps/management/commands/update_meps.py

```diff
@@ -382,6 +458,12 @@ def create_mep(mep_json):
     # print "     save mep modifications"
     mep.save()
 
+def json_parser_generator(json_file):
+    """Parse the json and yield one parltrack vote at the time
+     I need to parse the json file by hand, otherwise this eat way too much memory
+    """
```

Comment:
Yeah, we should maybe restart the python process at some point and seek to continue from where the previous one left the import to be sure that it won't block the data import and leave the db inconsisten.